pages/common/2to3.md

```diff
@@ -0,0 +1,28 @@
+# 2to3
+
+> Automated Python 2 to 3 code translation.
+> More information: <https://docs.python.org/3/library/2to3.html>.
+
+- Display changes that would be performed:
+
+`2to3 {{path/to/file.py}}`
+
+- Convert a Python 2 file to Python 3:
+
+`2to3 --write {{path/to/file.py}}`
+
+- Convert a specified Python 2 feature to Python 3:
+
+`2to3 --write {{file.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Convert all features except the specified ones to Python 3:
+
+`2to3 --nofix={{raw_input}} --fix={{print}} example.py`
```

Comment:
I just want to confirm: should this `--fix={{print}}` be `--nofix={{print}}`?

Reply:
@navarroaxel Yes.

Reply:
@navarroaxel Maybe it should be:
```
Convert all features except *raw_input* to Python 3:
```

Reply:
Yeah, please apply.

Reply:
@navarroaxel done @ 17badf8d78705416d4880d793d66d97697cb51c1